src/internal/views/nunjucks/gauging-stations/licences.njk

```diff
@@ -0,0 +1,223 @@
+    <div class="govuk-grid-column-full">
+      <h1 class="govuk-heading-l govuk-!-margin-bottom-1">
+        <span class="govuk-caption-l">{{[data.stations[data.stationID].catchmentName]}}</span>
+        {{title}}
+      </h1>
+      {% set metaData %}
+        {{ govukSummaryList({
+        classes: 'govuk-summary-list--no-border',
+        rows: [
+          {
+            key: {
+              text: "Grid reference",
+              classes: "meta-data__label"
+            },
+            value: {
+              text: [data.stations[data.stationID].gridReference],
+              classes: "meta-data__value"
+            }
+          } if data.stations[data.stationID].gridReference,
+          {
+            key: {
+              text: "Easting",
+              classes: "meta-data__label"
+            },
+            value: {
+              text: [data.stations[data.stationID].easting],
+              classes: "meta-data__value"
+            }
+          } if data.stations[data.stationID].easting,
+          {
+            key: {
+              text: "Northing",
+              classes: "meta-data__label"
+            },
+            value: {
+              text: [data.stations[data.stationID].northing],
+              classes: "meta-data__value"
+            }
+          } if data.stations[data.stationID].northing,
+          {
+            key: {
+              text: "WISKI ID",
+              classes: "meta-data__label"
+            },
+            value: {
+              text: [data.stations[data.stationID].wiskiId],
+              classes: "meta-data__value"
+            }
+          } if data.stations[data.stationID].wiskiId,
+          {
+            key: {
+              text: "Station reference",
+              classes: "meta-data__label"
+            },
+            value: {
+              text: [data.stations[data.stationID].stationReference],
+              classes: "meta-data__value"
+            }
+          } if data.stations[data.stationID].stationReference
+        ]
+        }) }}
+      {% endset %}
+
+    {{ govukDetails({
+      summaryText: "Station details",
+      html: metaData
+    }) }}
+
+  </div>
+</div>
+
+
+{% set tags = data.stations[data.stationID]['tags'] %}
+{% if tags.length %}
+  <div class="govuk-grid-row govuk-!-margin-bottom-6">
+    <div class="govuk-grid-column-full">
+      {{ govukButton({
+    text: "Create a water abstraction alert",
+    href: "licence-notices/send-a-water-abstraction-alert/select-the-type-of-alert?stationID=" + [data.stationID]
+  }) }}
+    </div>
+  </div>
+{% endif %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    <!-- Start tag-table.html: Licences linked to this monitoring station -->
+    {% block service %}
+        <!--setting the loop number to 0-->
+        {% set loopNumber = "" %}
+
+        <!--getting the licence data-->
+        {% set tags = data.stations[data.stationID]['tags'] %}
```

Comment:
Given that (I think) the page renders a single station, why does the view model contain multiple stations?

I think I would expect an array called `tags` to be output to the view model here.